Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Cursors/BaseCursor.cs

```diff
@@ -113,17 +113,17 @@ public bool ResizeCursorWithDistance
         private bool resizeCursorWithDistance = false;
 
         /// <summary>
-        /// The value, in angle, of expected cursor size in relation to Main Camera
+        /// The angular scale of cursor in relation to Main Camera, assuming a mesh with bounds of Vector3(1,1,1)
         /// </summary>
-        public float CursorAngularScale
+        public float CursorAngularSize
```

Comment:
If possible, please re-add a `public float CursorAngularScale` with the `Obsolete` tag (can just call directly into this property), so we don't break people currently calling this public property.

Reply:
By "can just call directly into this property" do you mean [Obsolete]CursorAngularScale will get and set cursorAngularSize?

Reply:
Yep! `cursorAngularSize` or `CursorAngularSize`, no need to add a separate backing field

Reply:
Thanks Kurtis, I just pushed an update. Let me know if what I did checks out.

Reply:
Looks great, thanks!